lib/policies/load-balancer/balancer.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const httpProxy = require('http-proxy');
+const logger = require('../../logger').gateway;
+
+const ConfigurationError = require('../../errors').ConfigurationError;
+const RoundRobin = require('./round-robin');
+
+const strategies = {
+  'round-robin': RoundRobin
+};
+
+const createStrategy = (name, proxy, endpoints) => {
+  const Strategy = strategies[name];
+  return new Strategy(proxy, endpoints);
+};
+
+module.exports = function (params, config) {
+  const endpoints = params.serviceEndpoints
+    .map(name => {
+      const endpoint = config.gatewayConfig.serviceEndpoints[name];
```

Comment:
yes, it should be good. 

HAProxy uses `algorithm`
http://cbonte.github.io/haproxy-dconv/configuration-1.4.html#4.2-balance

`strategy` or `algorithm` seems to be good. `method` is confusing with `HTTP method`